packages/pangea-sdk/pangea/request.py

```diff
@@ -57,7 +59,7 @@ def set_extra_headers(self, headers: dict):
         if isinstance(headers, dict):
             self._extra_headers = headers
 
-    def set_custom_user_agent(self, user_agent: str):
+    def set_custom_user_agent(self, user_agent: Optional[str]):
```

Comment:
Yes. Originally, `PangeaConfig.custom_user_agent` is optional, so to avoid mypy warning we should do it optional here too. We could do `PangeaConfig.custom_user_agent` and empty string by default and remove the Optional[] but that should be a breaking change.